packages/requisitions/src/CustomerRequisition/ListView/api.ts

```diff
@@ -21,7 +21,16 @@ const onRead =
     const requisitions = requisitionsGuard(result);
     const { nodes } = requisitions;
 
-    return { nodes, totalCount: requisitions.totalCount };
+    return {
+      nodes: nodes.map(requisition => ({
+        ...requisition,
+        color: requisition?.color ?? '#8f90a6',
+        orderDate: requisition?.orderDate
+          ? new Date(requisition.orderDate)
+          : null,
```

Comment:
parsing orderDate to a date and assigning a default `color` .. maybe it's better to just have a default color in the component 🤔

Reply:
yes, I like that idea. default colour in the component, makes things like this much easier.

and a date mapper? some other nifty way to map the dates?